k2/csrc/utils.cu

```diff
@@ -137,9 +137,10 @@ void RowSplitsToRowIds(ContextPtr &c, int32_t num_rows,
       int32_t avg_elems_per_row = (num_elems + num_rows - 1) / num_rows,
               threads_per_row = RoundUpToNearestPowerOfTwo(avg_elems_per_row),
               tot_threads = num_rows * threads_per_row;
-      int block_size = 256;
-      int grid_size = NumBlocks(tot_threads, block_size);
-      K2_CUDA_SAFE_CALL(RowSplitsToRowIdsKernel<<<grid_size, block_size, 0,
```

Comment:
Seems you reverted my changes in the last PR but actually you should not do this. `grid_size` is the number of blocks in a kernel and `block_size` is the number of threads in a block, so the current code you create has the wrong parameter order.

Reply:
And I prefer name `block_size` and `grid_size` (or `num_blocks`), as many cuda example code also use these two names, I think it's better if we can follow the same convention.

Reply:
Did it pass the unit test?

Reply:
I guess it may pass the tests for now as currently I don't add tests with hundreds of values  for `RaggedShape`, we may need to add those tests after get `RandomRaggedShape` tested.

Reply:
changed this here and other places. Compiled and tested.